### PR TITLE
Change status of a task in CoreValidListener

### PR DESCRIPTION
**Pull request by alexandrem-rte**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**
Updating the status of task during core valid process using Spring Cloud Stream

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*
Yes, the application.yml is updated. 

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by benrejebmoh — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -98,6 +100,18 @@
                 <artifactId>farao-glsk-document-api</artifactId>
                 <version>${farao.core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>gridcapa-task-manager-api</artifactId>
+                <version>${task-manager.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
```
[the remark this answers is not in the document]
Comment (WalAmeni) [reply pom.xml:109]: ok

File: gridcapa-core-valid-app/src/test/resources/application.yml [hunk 2]
```diff
@@ -4,21 +4,33 @@ spring:
...
-      - http://${MINIO_HOST:localhost}:${MINIO_PORT:9000}
\ No newline at end of file
+core-valid-runner:
+  bindings:
+    request:
+      destination: core-valid-requests
+      routing-key: "#"
+    response:
+      destination: core-valid-response
+      expiration: 60000
```
[the remark this answers is not in the document]
Comment (WalAmeni) [reply gridcapa-core-valid-app/src/test/resources/application.yml]: ok

File: gridcapa-core-valid-app/src/main/java/com/farao_community/farao/gridcapa_core_valid/app/UrlValidationService.java [hunk 3]
```diff
@@ -21,16 +21,16 @@
...
+    public UrlValidationService(UrlWhitelistConfiguration urlWhitelistConfiguration) {
+        this.urlWhitelistConfiguration = urlWhitelistConfiguration;
     }
 
     public InputStream openUrlStream(String urlString) {
-        if (securityProperties.getWhitelist().stream().noneMatch(urlString::startsWith)) {
+        if (urlWhitelistConfiguration.getWhitelist().stream().noneMatch(urlString::startsWith)) {
             StringJoiner sj = new StringJoiner(", ", "Whitelist: ", ".");
-            securityProperties.getWhitelist().forEach(sj::add);
+            urlWhitelistConfiguration.getWhitelist().forEach(sj::add);
```
[the remark this answers is not in the document]
Comment (WalAmeni) [reply gridcapa-core-valid-app/src/main/java/com/farao_community/farao/gridcapa_core_valid/app/UrlValidationService.java:33]: Done

**Review by benrejebmoh — APPROVED** [review]

**Comment by benrejebmoh** [comment]
I approve! But i don't have write access on this project to merge.

**Pull request merged**